test/kata6.joinNames.test.js

```diff
@@ -1,5 +1,21 @@
-const { joinNames } = require("../src");
+const { joinNames } = require('../src');
 
-describe("joinNames", () => {
-  test("returns string of names, seperated by commas and an ampersand", () => {});
+describe('joinNames', () => {
+	test('returns string of names, seperated by commas and an ampersand', () => {
+		expect(joinNames([])).toBe('');
+		expect(joinNames([{ name: 'Bart' }])).toBe('Bart');
+		expect(joinNames([{ name: 'Bart' }, { name: 'Lisa' }])).toBe('Bart & Lisa');
+		expect(
+			joinNames([{ name: 'Bart' }, { name: 'Lisa' }, { name: 'Maggie' }])
+		).toBe('Bart, Lisa & Maggie');
+		expect(
+			joinNames([
+				{ name: 'Bart' },
+				{ name: 'Lisa' },
+				{ name: 'Maggie' },
+				{ name: 'Homer' },
+				{ name: 'Marge' },
+			])
+		).toBe('Bart, Lisa, Maggie, Homer & Marge');
```

Comment:
I like that you've added tests for different array lengths. However, it would improve readability if you separated these out into their own test blocks. e.g.
```
 test("returns an empty string for empty arrays", () => {
    expect(joinNames([])).toBe("");
  });

  test("returns string of name for array of length 1", () => {
    expect(joinNames([{ name: "Bart" }])).toBe("Bart");
  });

  test("returns string names, separated by ampersand for array of length 2", () => {
    expect(joinNames([{ name: "Bart" }, { name: "Lisa" }])).toBe("Bart & Lisa");
  });

 test("returns string names, separated by ampersand for array of length 3", () => {
    expect(joinNames([{ name: "Bart" }, { name: "Lisa" }, { name: "Maggie" }])).toBe("Bart, Lisa & Maggie");
  });

  test("returns string of names, separated by commas and an ampersand, for long arrays", () => {
    expect(
      joinNames([
        {
          name: "Bart",
        },
        {
          name: "Lisa",
        },
        {
          name: "Maggie",
        },
        {
          name: "Homer",
        },
        {
          name: "Marge",
        },
      ])
    ).toBe("Bart, Lisa, Maggie, Homer & Marge");
  });
```